modules/client-configuration/pages/clients-centos.adoc

```diff
@@ -16,121 +16,73 @@ Traditional clients are not available on {centos}{nbsp}8.
 ====
 
 
-== Server Requirements
 
-Before you begin, check that your {productname} Server meets the requirements at xref:installation:hardware-requirements.adoc[].
+== Add Software Channels
 
-Taskomatic uses one CPU core, and requires at least 3072{nbsp}MB of RAM.
-To ensure that taskomatic has access to enough memory, open the [path]``/etc/rhn/rhn.conf`` configuration file, and add this line:
+Before you register {centos} clients to your {productname} Server, check that you have the {centos} product enabled, and the required channels are fully synchronized.
 
-----
-taskomatic.java.maxmemory=3072
-----
+ifeval::[{suma-content} == true]
 
-Restart Taskomatic:
-----
-systemctl restart taskomatic
-----
+The channels you need for this procedure are:
 
+[[centos-channels-wizard]]
+[cols="1,1,1", options="header"]
+.CentOS Channels - WebUI
+|===
+| OS Version | Base Channel   | Updates Channel
+| {centos} 6 | centos6-x86_64 | centos6-updates for x86_64
+| {centos} 7 | centos7-x86_64 | centos7-updates for x86_64
+| {centos} 8 | centos8-x86_64 | centos8-appstream for x86_64
+|===
```

Comment:
I think the product names are more relevant here. Especially for SUMA you get everything and do not need to setup anything additiponal to this.

Reply:
I'm wondering if using "products" makes things technically more correct, but potentially more confusing?